l2-frontend/src/ui-cards/DirectionsHistory/index.vue

```diff
@@ -90,11 +90,12 @@
               <a :href="row.pacs" title="Снимок" v-tippy target="_blank" class="btn btn-blue-nb" v-if="!!row.pacs">
                 <i class="fa fa-camera"/>
               </a>
-              <a :href="`/mainmenu/stationar#{%22pk%22:${row.parent.pk},%22opened_list_key%22:null,%22opened_form_pk%22:null,%22every%22:false}`"
-                 :title="'Принадлежит и/б: ' + [[row.parent.pk]] + '-' + [[row.parent.parent_title]]" v-tippy target="_blank" class="btn btn-blue-nb" v-if="!!row.parent.parent_is_hosp">
+              <a href="#" @click.prevent="role_can_use_stationar ? show_stationar(row.parent.pk) : null"
+                 :title="'Принадлежит и/б: ' + [[row.parent.pk]] + '-' + [[row.parent.parent_title]]" v-tippy class="btn btn-blue-nb"
```

Comment:
Кстати, лучше сделать это так:
```suggestion
                 :title="`Принадлежит и/б: [[${row.parent.pk}]]-[[${row.parent.parent_title}]]`" v-tippy class="btn btn-blue-nb"
```

Reply:
Сначала совсем не понял необходимость квадратных скобок, но зачем делать ненужные преобразования из массива в строку (которые к тому же не совсем корректны и случайны), если можно строку сделать

Reply:
Будет более производительно и понятно